src/Resources/contao/library/Contao/Combiner.php

```diff
@@ -212,11 +219,18 @@ public function getFileUrls()
 			{
 				$strPath = 'assets/' . $strTarget . '/' . str_replace('/', '_', $arrFile['name']) . $this->strMode;
 
+				// Load the existing file
+				if (file_exists(TL_ROOT . '/' . $strPath))
```

Comment:
> But the PR will conflict with the hotfix/4.4.3 changes anyway once they are merged.

OK, you just added the same functionality like this. But what about the `TL_ASSETS_URL`?